src/main.cpp

```diff
@@ -276,6 +292,32 @@ int main(int argc, char *argv[])
         channel->sendMessage(QByteArray("OPEN_SHADER\n") +
                              shaderName.c_str());
     }
+    if(hookSpecDef != "")
```

Comment:
I think this would be (very slightly) more logical as `if (!hookPayload.empty())`